eth-contracts/contracts/Governance.sol

```diff
@@ -592,6 +592,7 @@ contract Governance is InitializableV2 {
         _requireIsInitialized();
 
         require(msg.sender == address(this), ERROR_ONLY_GOVERNANCE);
+        require(stakingAddress == address(0), "Governance: Staking address already set");
         require(_stakingAddress != address(0x00), "Governance: Requires non-zero _stakingAddress");
```

Comment:
why don't other contracts require non-zero?

Reply:
Agreed, we will enforce that incoming _stakingAddress is not 0x00 in addition to standardizing order